gray/formatters/composite.py

```diff
@@ -4,10 +4,12 @@
 
 
 class CompositeFormatter(BaseFormatter):
-    _formatters: frozenset
+    # Use a list to guarantee order preserving, which can be important to
+    # ensure that the formatters are going to behave consistently.
+    _formatters: list
 
     def __init__(self, *formatters):
-        self._formatters = frozenset(formatters)
+        self._formatters = list(formatters)
```

Comment:
I think tuple should be used here, because formatters container must be immutable and tuple is preserve order too.

Reply:
Good catch, done.